pkg/uefivars/test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+// +build !release
```

Comment:
I'd put this in a separate package -- most of our users don't use a release tag. pkg/uefivars/vartest or something.